src/main/java/seedu/address/model/customer/Car.java

```diff
@@ -66,10 +66,7 @@ public String toString() {
 
     @Override
     public boolean equals(Object other) {
-        return other == this // short circuit if same object
-                || (other instanceof Car // instanceof handles nulls
-                && carBrand.equals(((Car) other).carBrand)
-                && carType.equals(((Car) other).carType)); // state check
+        return other == this; // Not checking carBrand or CarType since it shouldn't be unique
```

Comment:
um, i dont get why this is so? why just check objects for equality, same information should have same object. I think you should add another field like numberplate if you are worried about collisions. What you can do is if there is no explicit numberplate then just assign a random number to the numberplate (best random number is just the current timestamp in nanoseconds).

Reply:
We will shelve number plate away first, its not necessary